specs/protocol/isthmus/derivation.md

```diff
@@ -155,13 +155,13 @@ cast keccak $(cast concat-hex 0x000000000000000000000000000000000000000000000000
 Verify `data`:
 
 ```bash
-git checkout TODO
+git checkout 9436dba8c4c906e36675f5922e57d1b55582889e
 make build-contracts
 jq -r ".bytecode.object" packages/contracts-bedrock/forge-artifacts/OperatorFeeVault.sol/OperatorFeeVault.json
 ```
 
 This transaction MUST deploy a contract with the following code hash
-`TODO`.
+0x57dc55c9c09ca456fa728f253fe7b895d3e6aae0706104935fe87c7721001971.
```

Comment:
I'm getting `0x83d0e197ddf18068cfb3a7b07539aa1d3e4f30b39430edca78b49ae4f6be08d5` for the operator fee vault code hash.